### PR TITLE
feat: Added support for django3.2

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-2670

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
 - TOXENV=quality
 - TOXENV=pii-annotations
+- TOXENV=django22
+- TOXENV=django30
+- TOXENV=django31
+- TOXENV=django32
 matrix:
   include:
-  - python: 3.8
+  - python: '3.8'
```
Comment (mraarif) [inline .travis.yml:13]: we don't need this `include` section if we're running all tox envs against python 3.8

File: tox.ini [hunk 2]
```diff
@@ -25,13 +25,12 @@ sections = FUTURE,STDLIB,THIRDPARTY,DJANGO,DJANGOAPP,EDX,FIRSTPARTY,LOCALFOLDER
 ; D405 = Section name should be properly capitalized (numpy style)
 ; D406 = Section name should end with a newline (numpy style)
 ; D407 = Missing dashed underline after section (numpy style)
-; D408 = Section underline should be in the line following the section’s name (numpy style)
+; D408 = Section underline should be in the line following the section<E2><80><99>s name (numpy style)
```
Comment (mraarif) [inline tox.ini:28]: is this is an intentional change?

Comment (Jawayria) [reply tox.ini:28]: Just a mistake. Correcting it

**Review by mraarif — APPROVED** [review]

**Comment by Jawayria** [comment]
@edx/enterprise-markhors please review the PR

**Review by saleem-latif — APPROVED** [review]
Looks Good 👍🏼

**Comment by Jawayria** [comment]
> Looks Good 👍🏼

@saleem-latif Can you please merge the PR and release the package as I don't have access to merge it. Thanks

**Pull request merged**